R/ReportCard.R

```diff
@@ -20,8 +24,8 @@ ReportCard <- R6::R6Class( # nolint: object_name_linter.
     },
     #' @description Appends a table to this `ReportCard`.
     #'
-    #' @param table the appended table
-    #' @return invisibly self
+    #' @param table A (`data.frame` or object) that can be coerced into a table.
```

Comment:
What other kinds of objects are accepted?

Reply:
Its (`data.frame` or `rtables` or `TableTree` or `ElementaryTable` or `listing_df`) , Updating the types